schemas/documents/contributions/starter.tsx

```diff
@@ -233,6 +182,43 @@ export const starter = {
           return true
         }),
     },
+    {
+      name: 'showCompanyContactLink',
+      title: 'Show company contact information',
+      description: "Display a card on the template page with a link to your company's website.",
+      type: 'boolean',
+      defaultValue: false,
+    },
+    {
+      name: 'companyName',
+      title: 'Company name',
+      description: 'The name of the company that maintains this template.',
```

Comment:
Wouldn't name here just be the profile name of the creator?

Also, is "company" too specific. This can also apply to freelancers. Maybe "agency" is better?

Reply:
It wouldn't be profile name because profiles are just used for people and users can't create multiple profiles. So instead we're adding these "company" related fields, but I agree I think making it "agency" related fields is more broad _and_ more direct to the use case we're going for.

Reply:
It's tempting to make agency/organization more of a native thing and have this be a reference to a document type called "organization". 

Alternatively, how much of a lift would it be to just move the agency documents out of the admin studio and then allow for reference to them? None of them has incoming references, and it would be mostly to change the client in the frontend. This is slightly scope creepy, but feels more sustainable and correct, and would allow for them to reference showcases too.

Reply:
Yeah, we considered this when scoping but figured since we didn't need too out of an "Agency profile" at the moment, it would be easier to just add these couple fields